src/responses/user/notEmptyResponse.xml

```diff
@@ -1 +1 @@
initials><orgName>STFC</orgName><title>Mr</title><userNumber>4</userNumber></return><return><country>UNITED KINGDOM</country><deptName>ISIS</deptName><displayName>Unverified email Placeholder</displayName><email>unverified-user@example.com</email><establishmentId>7</establishmentId><familyName>Placeholder</familyName><firstNameKnownAs>Unverified email</firstNameKnownAs><fullName>Mr Unverified email Placeholder</fullName><givenName>Unverified email</givenName><initials>U</initials><orgName>STFC</orgName><title>Mr</title><userNumber>5</userNumber></return><return><country>UNITED KINGDOM</country><deptAcronym>SCD</deptAcronym><deptName>Scientific Computing</deptName><displayName>Carl Carlsson</displayName><email>Javon4@hotmail.com</email><establishmentId>5</establishmentId><familyName>Carlsson</familyName><firstNameKnownAs>Carl</firstNameKnownAs><fullName>Mr Carl Carlsson</fullName><givenName>Carl</givenName><initials>C</initials><orgName>STFC</orgName><title>Miss</title><userNumber>0</userNumber></return><return><country>UNITED KINGDOM</country><deptAcronym>SCD</deptAcronym><deptName>Scientific Computing</deptName><displayName>Carl Carlsson</displayName><email>Javon4@hotmail.com</email><establishmentId>5</establishmentId><familyName>Carlsson</familyName><firstNameKnownAs>Carl</firstNameKnownAs><fullName>Mr Carl Carlsson</fullName><givenName>Carl</givenName><initials>C</initials><orgName>STFC</orgName><title>Miss</title><userNumber>1</userNumber></return><return><country>UNITED KINGDOM</country><deptName>ISIS</deptName><displayName>test 3</displayName><email>unverified-user@example.com</email><establishmentId>7</establishmentId><familyName>Placeholder</familyName><firstNameKnownAs>Unverified email</firstNameKnownAs><fullName>Mr Unverified email Placeholder</fullName><givenName>Unverified email</givenName><initials>U</initials><orgName>STFC</orgName><title>Mr</title><userNumber>2</userNumber></return><return><country>UNITED KINGDOM</country><deptAcronym>SCD</deptAcronym><deptName>Scientific Computing</deptName><displayName>Nils</displayName><email>nils@ess.se</email><establishmentId>5</establishmentId><familyName>Nilsson</familyName><firstNameKnownAs>Nils</firstNameKnownAs><fullName>Mr Nils Nilsson</fullName><givenName>Nils</givenName><initials>N</initials><orgName>STFC</orgName><title>Mr</title><userNumber>3</userNumber></return><return><country>UNITED KINGDOM</country><deptAcronym>SCD</deptAcronym><deptName>Scientific Computing</deptName><displayName>David Dawson</displayName><email>david@teleworm.us</email><establishmentId>5</establishmentId><familyName>Dawson</familyName><fedId>abc12345</fedId><firstNameKnownAs>David</firstNameKnownAs><fullName>Mr David Dawson</fullName><givenName>David</givenName><initials>DD</initials><mobilePhone>08579366828</mobilePhone><orgName>STFC</orgName><status>Undergraduate Student</status><title>Mr</title><userNumber>6</userNumber></return></ns2:getBasicPeopleDetailsFromUserNumbersResponse></soap:Body></soap:Envelope>
```

Comment:
```suggestion
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"><soap:Body><ns2:getBasicPeopleDetailsFromUserNumbersResponse xmlns:ns2="http://webservice.UserOffice.stfc.com/"><return><country>UNITED KINGDOM</country><deptName>ISIS</deptName><displayName>Benjamin Beckley</displayName><email>ben@inbox.com</email><establishmentId>7</establishmentId><familyName>Beckley</familyName><firstNameKnownAs>Ben</firstNameKnownAs><fullName>Mr Benjamin Beckley</fullName><givenName>Benjamin</givenName><initials>B</initials><orgName>STFC</orgName><title>Mr</title><userNumber>4</userNumber></return><return><country>UNITED KINGDOM</country><deptName>ISIS</deptName><displayName>Unverified email Placeholder</displayName><email>unverified-user@example.com</email><establishmentId>7</establishmentId><familyName>Placeholder</familyName><firstNameKnownAs>Unverified email</firstNameKnownAs><fullName>Mr Unverified email Placeholder</fullName><givenName>Unverified email</givenName><initials>U</initials><orgName>STFC</orgName><title>Mr</title><userNumber>5</userNumber></return><return><country>UNITED KINGDOM</country><deptAcronym>SCD</deptAcronym><deptName>Scientific Computing</deptName><displayName>Carl Carlsson</displayName><email>Javon4@hotmail.com</email><establishmentId>5</establishmentId><familyName>Carlsson</familyName><firstNameKnownAs>Carl</firstNameKnownAs><fullName>Mr Carl Carlsson</fullName><givenName>Carl</givenName><initials>C</initials><orgName>STFC</orgName><title>Miss</title><userNumber>0</userNumber></return><return><country>UNITED KINGDOM</country><deptAcronym>SCD</deptAcronym><deptName>Scientific Computing</deptName><displayName>Carl Carlsson</displayName><email>Javon4@hotmail.com</email><establishmentId>5</establishmentId><familyName>Carlsson</familyName><firstNameKnownAs></firstNameKnownAs><fullName>Mr Carl Carlsson</fullName><givenName>Carl</givenName><initials>C</initials><orgName>STFC</orgName><title>Miss</title><userNumber>1</userNumber></return><return><country>UNITED KINGDOM</country><deptName>ISIS</deptName><displayName>test 3</displayName><email>unverified-user@example.com</email><establishmentId>7</establishmentId><familyName>Placeholder</familyName><firstNameKnownAs>Unverified email</firstNameKnownAs><fullName>Mr Unverified email Placeholder</fullName><givenName>Unverified email</givenName><initials>U</initials><orgName>STFC</orgName><title>Mr</title><userNumber>2</userNumber></return><return><country>UNITED KINGDOM</country><deptAcronym>SCD</deptAcronym><deptName>Scientific Computing</deptName><displayName>Nils</displayName><email>nils@ess.se</email><establishmentId>5</establishmentId><familyName>Nilsson</familyName><firstNameKnownAs>Nils</firstNameKnownAs><fullName>Mr Nils Nilsson</fullName><givenName>Nils</givenName><initials>N</initials><orgName>STFC</orgName><title>Mr</title><userNumber>3</userNumber></return><return><country>UNITED KINGDOM</country><deptAcronym>SCD</deptAcronym><deptName>Scientific Computing</deptName><displayName>David Dawson</displayName><email>david@teleworm.us</email><establishmentId>5</establishmentId><familyName>Dawson</familyName><fedId>abc12345</fedId><firstNameKnownAs>David</firstNameKnownAs><fullName>Mr David Dawson</fullName><givenName>David</givenName><initials>DD</initials><mobilePhone>08579366828</mobilePhone><orgName>STFC</orgName><status>Undergraduate Student</status><title>Mr</title><userNumber>6</userNumber></return></ns2:getBasicPeopleDetailsFromUserNumbersResponse></soap:Body></soap:Envelope>
```

I think we should keep the firstNameKnownAs for user 4 as Ben and remove it for user 1 to match the other files